assets/agenda/actions.ts

```diff
@@ -591,6 +591,7 @@ export function setAndUpdateNewItems(data: any) {
         }
 
         dispatch(updateItem(item));
+        dispatch(fetchItems());
```

Comment:
why does it need to refetch it? shouldn't it handle the changes via `updateItem`

Reply:
Nope, this is the case when events contain a related-planning item, so when we change the coverages in the events, here `(dispatch(updateItem(item)))` we get an event item with updated coverages rather than a planning item.
So, in the `planning only` view, associated planning items remain unchanged until we refetch.

Reply:
I thought the planning items are embedded there, but even if we need to refetch something, I suppose we don't need to refetch everything rigth?

Reply:
yep, you r right, so in this case, we can do something like fetch an event-related item via planning IDs?

Reply:
so we get just the planning item in the notification and not the actual event?

Reply:
we got a full event with an updated planning item in it, but it updates the event only, not its linked planning item state.

Reply:
so for this we can do something like this, so this updates its related planning item as well.
```.js
item?.planning_items?.map((data:any)=> {
    dispatch(fetchItem(data._id));
})

Reply:
shouldn't this be rather fixed in the reducer then?

Reply:
yep, it is more complicated because, in the events, we did not get full data of planning item, so first we need to refetch planning item then update it.